robot/EDA/resources/locators_50.py

```diff
@@ -215,6 +216,11 @@
         "next_save_button": "//div[contains(@class, 'inlineFooter')]/descendant::button[@title='Save']",
         "final_save_button": "(//span[@class=' label bBody' and text()='Save'])[3]/ancestor::button",
     },
+    "settings_health_check": {
+        "run_health_check_button": "//button[@title='{}']",
+        "health_check_header": "//h2[contains(@class, 'header')]/span[text()='{}']",
+        "last_run_date": "//button[@title='Run Health Check']/preceding::div[1]",
```

Comment:
@rhakeshj  will the position of this field remain the same?

Reply:
Yes . It is like a label to that button (just for a comparison) and it will always be in the immediate preceding div from the button